content/installation/net/usage/NetApplicationPoolFiltering.md

```diff
@@ -58,22 +58,26 @@ Use **Contrast .NET Logs**:
 
 ## Blacklist an Application Pool
 
-To disable the agent for a specific application, populate `ProcessBlacklist` with the appropriate application pool in *C:\Program Files\Contrast\dotnet\DotnetAgentService.exe.config*:
+To disable the agent for a specific application, populate `agent.dotnet.app_pool_blacklist` with the appropriate application pool in *C:\ProgramData\Contrast\dotnet\contrast_security.yaml*:
 
 ```
 <!--Comma-separated list of application pools ignored by Contrast-->
```

Comment:
Maybe switch to yaml comment style here?

Reply:
Good idea. Switched.